scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/language/json/JsonParser.scala

```diff
@@ -54,8 +55,10 @@ object JsonParser extends ExpressionParser {
 
 }
 
-case object JsonExpressionTypingInfo extends ExpressionTypingInfo {
+case class JsonExpressionTypingInfo(json: Json) extends ExpressionTypingInfo {
+
+  override val typingResult: TypingResult = {
+    Typed.fromInstanceWithNullsAsUnknowns(FromJsonSimpleDecoder.jsonToAny(json))
```

Comment:
The another difference from standard `Typed.fromInstance` invocation that we should  apply here, is that we don't want to have inferred values in types. It is because the user provides not a value that they wish to type but an example, so the type should be wider. You can achieve that using `.withoutValue`. I'm thinking that we don't even need `fromInstanceWithNullsAsUnknowns` method. The `.withoutValue` should generalize `TypedNull` to `Unknown` (Any)